docker/README.md

```diff
@@ -0,0 +1,14 @@
+### Building
+
+This image requires to specify following build arguments:
```

Comment:
Perhaps add some more documentation here?   What is this image?  How does one use it?   How/when does it get updated?

Reply:
Not sure about answer for _How/when does it get updated?_, ideally it should be build and published as post build step after a release of Emscripten gets published. For that I will need some helping hand.

Reply:
For now maybe lets just include instructions on how to update the image?  And how to use it.

Reply:
Maybe a link to the image in the docker registry and how to update it?

Reply:
Added helper makefile + extended info how to build, push and tag this image